CMakeLists.txt

```diff
@@ -61,7 +62,7 @@ ELSEIF(NOT CMAKE_SYSTEM_NAME MATCHES "^(Darwin|Linux|Android|Windows)$")
 ENDIF()
 
 # ---[ Download deps
-IF(NOT DEFINED CLOG_SOURCE_DIR)
+IF(NOT DEFINED CLOG_SOURCE_DIR AND NOT USE_SYSTEM_LIBS)
```

Comment:
Please wrap the whole block that downloads dependencies into `IF(NOT XNNPACK_USE_SYSTEM_LIBS)`